src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -64,6 +85,29 @@ public void arcadeDrive(double translate, double rotate) {
 
         setRawPower(left, right);
     }
+    
+    public void kinematicsDrive(double linearVelocity, double angularVelocity) {
+        var chassisSpeed = new ChassisSpeeds(linearVelocity, 0.0, angularVelocity); // 0.0 = no y directional
+        wheelSpeeds = kinematics.toWheelSpeeds(chassisSpeed);
+        
+        double leftPower = getLeftSpeedVoltage(wheelSpeeds)/leftMaxVoltage;
+        double rightPower = getRightSpeedVoltage(wheelSpeeds)/rightMaxVoltage;
+
+        leftMaster.setVoltage(leftPower);
+        rightMaster.setVoltage(rightPower);
+    }
+
+    public double getLeftSpeedVoltage(DifferentialDriveWheelSpeeds speeds) {
```

Comment:
I think we can remove these methods and put their logic directly in the `kinematicsDrive` method. They're mostly duplicates of each other, and just wrapping around the single core line of `feedforward.calculate()`